tests/unit/test__http.py

```diff
@@ -32,6 +32,9 @@ def _get_target_class():
         return Connection
 
     def _make_one(self, *args, **kw):
+        if "api_endpoint" not in kw:
+            kw["api_endpoint"] = "https://storage.googleapis.com"
```

Comment:
Shouldn't matter for testing, but let's be accurate.

```suggestion
            kw["api_endpoint"] = "https://bigquery.googleapis.com"
```